local-cluster/Main.hs

```diff
@@ -108,5 +121,6 @@ data CWalletConfig = CWalletConfig
   , adaAmount  :: Integer
   , lvlAmount  :: Integer
   , numUtxos :: Int
+  , dirWorking :: Maybe FilePath
```

Comment:
Maybe name it also something like `workDir` or `clusterWorkingDir`

Reply:
Changed to `workDir`, since `clusterWorkingDir` is being used by `PlutipConfig`.